app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>A modern languages teacher can  be a rewarding way to use your language knowledge, while opening young minds to the languages and cultures around the world.</p>
+  <p>Teacher training is available in 18 modern languages with the most popular being French, German and Spanish in key stage 2 and 3.</p>
+  <p>Tax-free  bursaries of £25,000 or scholarships of £27,000 are available for eligible trainee modern languages teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Empower your pupils",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+   <p>"Language learning empowers learners, opens doors, and creates valuable skills for the next generation."</p>
```

Comment:
@caitlin-reidy this quote doesn't really make sense. And we talk about opening young minds in the intro. Might be better to lose it altogether for now? Or if not reword to Learning a language creates valuable skills and empowers the next generation.

Reply:
@maxineCl i've change the quote if that works better?